spec/factories/messages.rb

```diff
@@ -4,6 +4,7 @@
   factory :message do
     created_at { Time.zone.now }
     unknown_content { false }
+    text { 'Am I an incoming message?' }
```

Comment:
Actually, I changed it to just be a `Faker::Lorem.sentence` since if we override the trait `with_sender` in the test then it would not be an incoming message, but an outgoing one.
I still like never having a `nil` text. A little surprised we don't validate that this cannot happen.